docs/README.md

```diff
@@ -97,7 +106,7 @@ curl -sfL http://get.kubearmor.io/ | sudo sh -s -- -b /usr/local/bin
 
 * **Discovery Engine Installation:**
 
-1. Download the [latest release](https://github.com/accuknox/discovery-engine/releases) of Discovery Engine
+1. Download the [latest release ](https://github.com/accuknox/discovery-engine/releases){:target="_blank"}{: .externalLink} of Discovery Engine
```

Comment:
> [latest release ]

[latest release]

Reply:
The extra space is intentional. It will separate the linked text from the launch glyph.